test/unit/specs/components/Identity/createComponent.spec.js

```diff
@@ -122,22 +122,22 @@ describe("Identity::createComponent", () => {
 
   it("syncIdentity syncs identities", () => {
     const identities = { type: "identities" };
-    component.commands.syncIdentity.run({ identities });
-    expect(identityManager.sync).toHaveBeenCalledWith(identities);
+    return component.commands.syncIdentity.run({ identities }).then(result => {
+      expect(identityManager.sync).toHaveBeenCalledWith(identities);
+      expect(result).toBeUndefined();
+    });
   });
 
   it("getIdentity command should make a request when ecid is not available", () => {
-    let ecid;
+    const onResolved = jasmine.createSpy("onResolved");
     component.commands.getIdentity
       .run({ namespaces: ["ECID"] })
-      .then(identities => {
-        ecid = identities.ECID;
-      });
+      .then(onResolved);
```

Comment:
I switched this to use a spy because I think it's easier to read and, if something fails (like `identity` is supposed to be `identities`, the jasmine output is easier to understand.